sklearn/base.py

```diff
@@ -259,7 +259,7 @@ def set_params(self, **params):
         self
         """
         if not params:
-            # Simple optimisation to gain speed (inspect is slow)
+            # Simple optimization to gain speed (inspect is slow)
```

Comment:
`optimisation` is valid in the UK, yet I agree our codebase is more US-oriented.
+1